### ISSUE TITLE
bug: Ionic date picker not accessible/usable with talkback

### ISSUE THREAD

**Issue by mastergogo** [ionic-team/ionic-framework#21810]
We are using ionic [date and time picker](https://ionicframework.com/docs/api/datetime) and when checked with Talkback, it has been observed that date picker is very difficult to use from screen reader (talkback) when the dropdown opens, it does not announce whether the focus is on DD, MM or YY, to fill the date user have to keep swiping until they reach the desired DD, then it does not announce when it switches to MM or YY,  which makes it extremely time-consuming and hard to use, I checked the date picker which comes directly in the calendar app of Android OS is way more accessible and useable. let me know if more info is needed. I checked this in Samsung galaxy M30 with Android 10.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Your issue has been added to our `ion-datetime` issue tracker under `datetime not working well with accessibility settings`: https://github.com/ionic-team/ionic-framework/issues/16630
